web-devtools/src/app/(main)/dispute-template/CustomContextInputs.tsx

```
@@ -0,0 +1,82 @@
+    300,
+    [customContext]
+  );
+
+  return requiredVariables.length ? (
+    <Container>
+      <WithHelpTooltip tooltipMsg="These are additional variables required by the data mapping to be passed as initial context. Please ignore the variables that will come from the result of the preceeding data mappings">
+        <Header>Additional Context</Header>
+      </WithHelpTooltip>
+      {requiredVariables.map((variable, index) =>
+        DisputeRequestParams.includes(variable) ? null : (
+          <InputContainer id={`${variable}-${index}`}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)